tests/e2e/env/config/use-config.js

```diff
@@ -1,5 +1,64 @@
 const jestConfig = require( './jest.config.js' );
```

Comment:
> Since all of these functions are identical I wonder if we should refactor them?
> In fact, if all we're doing is merging arrays,

Some are arrays. Some are objects. While not identical, it still wouldn't be too complex to refactor them.